### ISSUE TITLE
undefined method `convert_inline_menu'

### ISSUE THREAD

**Issue by dismine** [slonopotamus/asciidoctor-fb2#12]
Hi.

I have tried to use your tool to prepare fb2 version of my book but got this error: 

```bash
undefined method `convert_inline_menu' for #<Asciidoctor::FB2::Converter:0x0000556df3ea73b8>
Did you mean?  convert_inline_image
  Use --trace for backtrace
```

```bash
asciidoctor-fb2 -v
Asciidoctor FB2 0.1.0 using Asciidoctor 2.0.10 [https://asciidoctor.org]
Runtime Environment (ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-linux-gnu]) (lc:UTF-8 fs:UTF-8 in:UTF-8 ex:UTF-8)
```

**Comment by slonopotamus** [comment]
This was fixed in 0.2.0. Please, note that asciidoctor-fb2 is in very early development, so you might need to report several more bugs before it will work for you :D

**Comment by dismine** [comment]
Thanks for the quick fix!
